### ISSUE TITLE
amazon_transcribe.exceptions.BadRequestException: A complete signal was sent without the preceding empty frame.

### ISSUE THREAD

**Issue by tyhunt99** [awslabs/amazon-transcribe-streaming-sdk#21]
I am having issues getting the basic script working. I am using a very small local wav file of about 17 seconds. I am on python 3.9 and the following package versions:

```
amazon-transcribe        0.2.0
awscrt                             0.9.16
```

I did at first have an error during read:
```
Traceback (most recent call last):
  File "/Users/thunt/aws-crt-python/awscrt/io.py", line 554, in _read_into_memoryview
    return self._stream.readinto1(m)
TypeError: read() should return bytes
```

This I was able to get around (may not actually be a work around), by changing `BufferableByteStream` in `structures.py` to return an empty byte `b""`

But this then resulted in the new error `amazon_transcribe.exceptions.BadRequestException: A complete signal was sent without the preceding empty frame.`

Another no,  I have the repo installed locally due to #20. Which allowed me to install them.

Any insight would be appreciated.

**Comment by Downchuck** [comment]
That read issue seems to be in 3.8 as well, though 3.7 works.

**Comment by nateprewitt** [comment]
I believe this was unintentionally resolved in v0.3.0 which we just released. If anyone in this thread can confirm they're no longer experiencing the issue after upgrading we can close this ticket.

**Comment by nateprewitt** [comment]
Resolving since we haven't heard further feedback and I'm unable to reproduce. Please feel free to ping or reopen this ticket if there's further evidence of this not working in v0.3.0. Thanks!
